hpat/hiframes/pd_dataframe_ext.py

```diff
@@ -1554,7 +1554,7 @@ def generic(self, args, kws):
         df = args[0]
         # TODO: ignore non-numerics
         # get series prod output types
-        dtypes = tuple(hpat.hiframes.pd_series_ext.SeriesAttribute.resolve_prod(
+        dtypes = tuple(numba.typing.arraydecl.ArrayAttribute.resolve_prod(
```

Comment:
What issue you trying to solve by this change?

Reply:
`SeriesAttribute` have no `resolve_prod()` because it was not copied from `ArrayAttribute`.
[Here](https://github.com/IntelPython/hpat/blob/94cb552f3e2c11fe62523661fe6fa2f1af2d34da/hpat/hiframes/pd_series_ext.py#L992
) I added `'resolve_prod'` into `_not_series_array_attrs` and it prevents copying of corresponding method from `ArrayAttribute` to `SeriesAttribute` [here](https://github.com/IntelPython/hpat/blob/94cb552f3e2c11fe62523661fe6fa2f1af2d34da/hpat/hiframes/pd_series_ext.py#L998).

Reply:
The issue is exception with 'attribute not found'